typescript/README.md

```diff
@@ -0,0 +1,94 @@
+# @cooklang/cooklang-ts
+
+A TypeScript/JavaScript library for parsing CookLang recipes, built with Rust and WebAssembly.
+
+> [!NOTE]  
+> Package is under development. What exists here may not be published to NPM yet.
+
+## Prerequisites
+
+This project uses Rust compiled to WebAssembly, so you'll need to set up the Rust toolchain properly before development.
+
+### Required Tools
+
+1. **Rust via rustup** (recommended over system package managers)
+2. **wasm-pack** 
+3. **Node.js**
+
+### Setup Instructions
+
+#### 1. Install Rust (via rustup)
+
+**Important:** Use rustup rather than system package managers (Homebrew on macOS, apt on Ubuntu, etc.) for better toolchain management.
+
+If you have Rust installed via a package manager, consider uninstalling it first:
+```bash
+# macOS (Homebrew)
+brew uninstall rust
+
+# Ubuntu/Debian
+sudo apt remove rustc
+
+# Arch Linux
+sudo pacman -R rust
+```
+
+Install Rust using rustup. Platform specific instructions at [rustup.rs](https://rustup.rs/).
+
+After installation, reload your shell environment:
+```bash
+# Unix-like systems (macOS, Linux)
+source ~/.cargo/env
+
+# Windows (restart your terminal or run)
+# The installer usually handles PATH automatically
+```
```

Comment:
There isn't value in our repo needing to maintain this content. We can point at the docs from Rust and their installation instructions. This saves us from needing to update our docs if any Rust specifics (and all OS considerations) change.